src/main/java/ch/njol/skript/patterns/TypePatternElement.java

```diff
@@ -264,7 +267,7 @@ else if ((flagMask & SkriptParser.PARSE_EXPRESSIONS) == 0)
 		return stringBuilder.append("%").toString();
 	}
 
-	private ExprInfo getExprInfo() {
+	public ExprInfo getExprInfo() {
```

Comment:
```suggestion
	private ExprInfo getExprInfo() {
```
We can expose this in a future PR if necessary